### DISCUSSION TITLE
remon이 오늘 회고때 좋은점 첫번째로 이야기한 바벨 절대경로가 설정 편의성은 어떻게 하면 저도 느낄 수 있나여?

### DISCUSSION THREAD

**Discussion by gygy7151** [EasyAndBeauty/DailyReceipt-FE-WEB#22] [Q&A]
바벨이 오디에서 사용되었는지 조차 모르고 있었어여
혹시.. 해당 코드는 어디서 보면 되는건지 알려 주실 수 있나여?

**Comment by parkjooyoung0608** [comment]
어먹...제가 잘 못 얘기했어요..!ㅠㅠ 바벨이 아니고 jsconfig.json에서 절대경로 설정한 것에 대한 얘기였습니다!
